src/ClusterBootstrap/params.py

```diff
@@ -91,6 +93,7 @@
     "storagemanager": "storagemanager",
     "repairmanager": "repairmanager",
     "repairmanageretcd": "repairmanageretcd",
+    "repairmanageragent": "repairmanageragent",
```

Comment:
where is this value rendered?

Reply:
Also in params.py, default_kube_labels_by_node_role["worker"] need to have "repairmanageragent": "active"

Reply:
It is rendered in the dockerfile: https://github.com/microsoft/DLWorkspace/blob/775b8f61e74ffb240dd12b0d80616bfb1ba90f4a/src/ClusterBootstrap/services/repairmanageragent/repairmanageragent.yaml#L25

Reply:
@Anbang-Hu , I noticed that pod yaml under services have different way to specify the image, e.g., storagemanager & restfulapi use {{cnf[]}} and have default value in params.py, while watchdog hard-coded the image url. I suggest later we refactor and formalize this.

Reply:
@YinYangOfDao just FYI I used jobmanager service as a reference for repairmanager service so this is why I have done this way. agree that there should be a formal way to do this for all services.